src/side-navigation/styled-components.js

```diff
@@ -69,6 +70,16 @@ export const StyledNavItemElement = styled<SharedPropsT>('div', props => {
   const paddingSuffixDir: string =
     $theme.direction === 'rtl' ? 'paddingLeft' : 'paddingRight';
 
+  let cursor = $selectable ? 'pointer' : 'default';
+  let color = $active ? colors.primary : null;
+  let hoverColor = $selectable ? colors.primary : null;
+
+  if ($disabled) {
+    cursor = 'not-allowed';
```

Comment:
Personally I would keep it as `default`. `not-allowed` seems too aggressive and we don't use it for disabled elements?

Reply:
we've discussed this offline, but for prosperity's sake: turns out ,we use it in quite a few places, so we've decided to keep things as is https://github.com/uber/baseweb/search?q=not-allowed&unscoped_q=not-allowed